src/release/main.d

```diff
@@ -48,6 +48,16 @@ void main ( string[] params )
 
     checkOAuthSetup();
 
+    try
+    {
+        import release.gitHelper;
+        getRemote(getUpstream());
+    }
+    catch (Exception exc)
+    {
+        writefln("Warning: %s", exc.msg);
```

Comment:
It can still do the local only actions. It just can't push them then. Maybe you feel like doing that yourself or maybe you wanted local only etc..